WebViewControl/ChromiumBrowser.cs

```diff
@@ -1,9 +1,16 @@
-﻿using Xilium.CefGlue;
-
-namespace WebViewControl {
-
-    internal partial class ChromiumBrowser {
-
-        internal CefBrowser GetBrowser() => UnderlyingBrowser;
-    }
-}
+﻿using Xilium.CefGlue;
+using Xilium.CefGlue.Avalonia;
+
+namespace WebViewControl {
+
+    class ChromiumBrowser : AvaloniaCefBrowser {

```

Comment:
this file is different from the original (Avalonia)

Reply:
this is a merge of ChromiumBrowser.cs and ChromiumBrowser.Avalonia.cs. Do you want me to split it again?